src/locales/fr/translation.json

```diff
@@ -113,10 +140,13 @@
   "of severe": "de sévère",
   "Ooops! This page does not exist": "Oups! Cette page n'existe pas",
   "Oops! Something went wrong": "Oups! Quelque chose a mal tourné",
-  "% of total": "% Du total",
+  "peak critical": "Pic critique",
+  "peak overflow": "Débordement",
+  "peak severe": "Pic grave",
   "provided parameter estimates.": "fourni des estimations de paramètres.",
   "Reduction of transmission through mitigation measures over time. Different presets with variable degree of reduction can be selected from the dropdown.": "Réduction de la transmission grâce à des mesures d'atténuation au fil du temps. Différents préréglages avec un degré de réduction variable peuvent être sélectionnés dans la liste déroulante.",
-  "This table summarizes the assumptions on severity which are informed by epidemiological and clinical observations in China. The first column reflects our assumption on what fraction of infections are reflected in the statistics from China, the following columns contain the assumption on what fraction of the previous category deteriorates to the next. These fields are editable and can be adjusted to different assumptions. The last column is the implied infection fatality for different age groups.": "Ce tableau résume les hypothèses de gravité qui sont basées sur des observations épidémiologiques et cliniques en Chine. La première colonne reflète notre hypothèse sur la fraction d'infections reflétée dans les statistiques de la Chine, les colonnes suivantes contiennent l'hypothèse sur la fraction de la catégorie précédente qui se détériore dans la suivante. Ces champs sont modifiables et peuvent être ajustés selon différentes hypothèses. La dernière colonne est la mortalité implicite d'infection pour différents groupes d'âge.",
+  "This table summarizes the assumptions on severity which are informed by epidemiological and clinical observations in China. The first column reflects our assumption on what fraction of infections are reflected in the statistics from China, the following columns contain the assumption on what fraction of the previous category deteriorates to the next. These fields are editable and can be adjusted to different assumptions. The last column is the implied infection fatality for different age groups.": "Ce tableau résume les hypothèses de gravité qui sont basées sur des observations épidémiologiques et cliniques en Chine. La première colonne reflète notre hypothèse sur la fraction d'infections reflétée dans les statistiques de la Chine, les colonnes suivantes contiennent l'hypothèse sur la fraction de la catégorie précédente qui se détériore dans la suivante. Ces champs sont modifiables et peuvent être ajustés selon différentes hypothèses. La dernière colonne est la mortalité implicite d'infection pour différentes tranches d'âge.",
```

Comment:
*les colonnes suivantes contiennent l'hypothèse sur la fraction de la catégorie précédente qui se détériore dans la suivante*: that sounds odd. I would rephrase it more drastically. Maybe *les colonnes suivantes représentent quelle fraction des cas de la catégorie précédente se sont aggravés*?